source/develop/release-management/features/network/ipv6_subnets.html.md

```diff
@@ -100,26 +138,98 @@ The ovirt-provider-ovn provides an opinionated subset of OpenStack's networking
 * dhcpv6-stateful
 * dhcpv6-stateless
 
-Given the priorities stated in [Objectives](#objectives), the ovirt-provider-ovn should only accept the 'dhcpv6-stateful' configuration, and fail for the other 2 options. When not provided, this parameter will of course default to the only accepted value.
+Given the priorities stated in [Objectives](#objectives), the ovirt-provider-ovn will accept either the 'dhcpv6-stateful' or 'dhcpv6-stateless' configuration, while *slaac* is not yet supported. When not provided, this parameter will default to dhcpv6-stateful, since a router is required for the stateless option.
+
+Also, only the **ipv6_address_mode** option will be implemented, since their
+behaviors kind of overlap - e.g. *slaac* and *dhcpv6-stateless* **require** RAs
+to configure their IP addresses, so, no point in having a separate setting to
+configure those.
 
 ## Routing
 
 Routing among IPv6 subnets is seemlessly provided by an OVN logical router
 entity.
 
-Since the scope of the feature is just to implement stateful DHCPv6, no router
-changes are required.
+OVN logical routers are also required whenever the 'slaac' or
+'dhcpv6-stateless' options are configured on the subnet, since they are what
+generates the RA messages that clients require to configure their IP addresses.
 
-If, in a given future, stateless address auto-configuration becomes a goal,
-the logical router ports have to be configured, setting its
-**ipv6_ra_configs:address_mode** to 'slaac'.
+Just creating the logical router is indeed not enough; each subnet that is
+configured to operate statelessly - either 'slaac' or dhcpv6_stateless -
+requires that very same subnet to be connected to at least one OVN logical router.
 
+That will create OVN logical flows that are responsible for generating the
+Router Advertisements messages the clients require to configure their IP addresses.
+These RA messages are generated as replies to the Router Solicitation messages.
+
+It is also possible to send those periodically, but I see no reason to do so.
 The above change would be implemented in the following way:
+
 1. when a subnet is created, it's **ipv6_address_mode** parameter is stored in
 the subnet's external ids
 2. when that subnet is attached to a router, its **ipv6_ra_configs:address_mode**
 should be set from the subnet's external ids address mode value.
 
```

Comment:
very advanced and can be skipped: is NAT still required, or can we have internet access for ovn networks with IPv6?

Reply:
Good question, to which I have no proper answer.

I do not know for sure, but from what I'm reading on the documentation, I get the impression that NAT is required.

I think that requires an RFE of its own, but I'll investigate that a bit.

Reply:
Been kind of investigating this, and from my understanding, NAT is *not* needed.

Otherwise, document [0] cannot be right - it claims connection to the internet, **but** we do not have NAT implemented.

I want to try this out with an ipv6 subnet running on top of a physnet connecting to an ipv6 external network. Either I'm not understanding something (which is a possibility), or, it's achievable.

[0] - https://github.com/oVirt/ovirt-provider-ovn/blob/master/docs/README-routing.md#external-gateways

Reply:
I see no way implementing stateless dhcpv6 - and using that as default - will clash with internet access.

If you have any indication to think so, let me know.

Otherwise, I'd prefer to have that as a separate study and RFE, otherwise this one will go on indefinitely.